src/DynamoCoreWpf/Views/Core/NoteView.xaml

```diff
@@ -76,6 +76,14 @@
                         <ToolTip Content="Select a node to pin to this note" Style="{StaticResource GenericToolTipLight}"/>
                     </MenuItem.ToolTip>
                 </MenuItem>
+                <MenuItem Name="UnpinToNode"
+                          Header="{x:Static p:Resources.ContextUnpinFromNode}" 
+                          Command="{Binding Path=UnpinFromNodeCommand}"
+                          ToolTipService.ShowOnDisabled="True">
+                    <MenuItem.ToolTip>
+                        <ToolTip Content="Unpin this note from the node" Style="{StaticResource GenericToolTipLight}"/>
```

Comment:
Tooltip content need to be added as resource string as well for localization

Reply:
@QilongTang tooltip was added as resource in the next commit:
https://github.com/DynamoDS/Dynamo/pull/12578/commits/e87e3c0711eddfeeb5c8c780be72c352ed5e22fe